libs/@blockprotocol/type-system/crate/Makefile.toml

```diff
@@ -66,8 +66,9 @@ args = ["build", "--profile", "${CARGO_MAKE_CARGO_PROFILE}", "@@split(CARGO_BUIL
 # TODO: add profile
 # TODO: consider moving away from wasm-pack and using wasm-bindgen directly so we can use weak-refs
```

Comment:
Can you remove this TODO _in a follow-up PR_ when looking into weak-refs like I pinged you about?